dbuild/__init__.py

```diff
@@ -96,7 +96,7 @@ def docker_build(build_dir, build_type, source_dir='source', force_rm=False,
                  docker_url='unix://var/run/docker.sock', dist='ubuntu',
                  release='trusty', extra_repos_file='repos',
                  extra_repo_keys_file='keys', build_cache=True, proxy="",
-                 build_owner=None):
+                 build_owner=None, **kwargs):
```

Comment:
Can you add a quick note to the documentation about this? All the other args are documented.